### PR TITLE
Update content on supplier details page

### PR DESCRIPTION
**Pull request by Wynndow**
Some suppliers didn't realise the gravity of changing these details.
This updated content should help to clarify.

### CODE REVIEW

File: app/templates/suppliers/details.html [hunk 1]
```diff
@@ -188,11 +188,12 @@
       <div class='column-two-thirds'>
         <form method="POST" action="{{ url_for('.confirm_supplier_details') }}">
           <input type="hidden" name="csrf_token" value="{{ csrf_token() }}"/>
-          <p>Once you confirm your information you'll need to contact support to change your:</p>
+          <p>Once you confirm your information you'll need to contact support to correct a mistake in your:</p>
```
Comment (samuelhwilliams) [inline app/templates/suppliers/details.html]: Should this be `contact CCS`?

**Comment by katstevens** [comment]
Is there a functional test PR to go along with this? Might make sense to review them together?

**Comment by Wynndow** [comment]
Yep, I'm putting it together now, about to push

File: app/templates/suppliers/details.html (hunk 1 above)
Comment (Wynndow) [reply app/templates/suppliers/details.html]: No, it'll still need to come through support. And they'll get the email address when they actually click the link on the thing when they want to update it.

Comment (samuelhwilliams) [reply app/templates/suppliers/details.html]: Right, but it's a CCS email address.

Eh, ok 🤷‍♂️

**Pull request merged**